static/src/js/util/files/parseUserAgent.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Get the operating system of the client
+ * @param {String} userAgent - A string containing the navigator.userAgent from browser
+ * @return {String} The operating system from the client as a string being returned
```

Comment:
Minor thing but I think this would be:

```suggestion
 * @return {(String|Null}} The operating system from the client as a string being returned
```